### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by pzakha**
The `delphix-kernel` package has been pulled outside of `delphix-platform`.

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# The following packages explicitly state the version of the linux
```
Comment (prakashsurya) [inline debian/rules]: Is this comment correct? It seems misleading, or maybe I'm confused.

File: debian/rules [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# kernel to be installed, as well as any packages that are kernel
+# specific (e.g. tools like perf, that are tied to the kernel version).
+# Any additional tools that are not specific to the kernel version
+# should be included in the "tools.list.chroot" file.
+#
+
+%:
+	dh $@
+
+override_dh_auto_build:
```
Comment (prakashsurya) [inline debian/rules]: Is this needed? We don't have a `Makefile` in this repository, so I don't think this or the `override_dh_auto_test` stuff below is needed, but maybe I'm wrong...

**Review by prakashsurya — APPROVED** [review]
Looks reasonable to me

File: debian/rules (hunk 1 above)
Comment (pzakha) [reply debian/rules]: Oh yeah, I've extracted it from the `delphix-package` project but forgot to update the comment. I'll do that.

**Pull request merged**